examples/github-team/main.tf

```diff
@@ -6,22 +6,23 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 resource "github_repository" "repository" {
-  name = var.a-repository-name
+  name = "terraform-github-team-module-test-repository-1"
```

Comment:
This is not a test please use better example names from real world ..

Reply:
makes sense, fixed in cf14787